api/config_entry_routes.go

```diff
@@ -225,7 +225,7 @@ type RetryFilter struct {
 	NumRetries            *uint32
 	RetryOn               []string
 	RetryOnStatusCodes    []uint32
-	RetryOnConnectFailure *bool
```

Comment:
Why did we remove the pointer?

Reply:
`TestHTTPRouteRetryAndTimeout` failed with a panic.